relay-spans/src/span.rs

```diff
@@ -0,0 +1,350 @@
+use std::str::FromStr;
+
+use chrono::{TimeZone, Utc};
+use serde::Deserialize;
+use serde_repr::Deserialize_repr;
+
+use relay_event_schema::protocol::{Span as EventSpan, SpanId, SpanStatus, Timestamp, TraceId};
+use relay_protocol::{Annotated, Object, Value};
+
+use crate::status_codes;
+
+#[derive(Clone, Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Span {
+    pub trace_id: String,
+    pub span_id: String,
+    #[serde(default, skip_serializing_if = "String::is_empty")]
+    pub trace_state: String,
+    #[serde(default, skip_serializing_if = "String::is_empty")]
+    pub parent_span_id: Option<String>,
```

Comment:
Could not find a formal OTel schema definition, but it seems that `parent_span_id` is an empty string for root spans: https://opentelemetry.io/docs/concepts/signals/traces/#spans

Reply:
I intended to remove the `Option` as you suggested before, so I'll fix this. Are you also suggesting to remove the `skip_serializing_if` in case the string is empty so we propagate the empty value in case of a root span?

Reply:
> Are you also suggesting to remove the skip_serializing_if in case the string is empty so we propagate the empty value in case of a root span?

No, `skip_serializing_if` is fine here IMO.